support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton.tsx

```diff
@@ -0,0 +1,167 @@
+import { css } from '@emotion/core';
+import { Button } from '@guardian/src-button';
+import { neutral, space } from '@guardian/src-foundations';
+import { PaymentRequestButtonElement } from '@stripe/react-stripe-js';
+import React from 'react';
+import GeneralErrorMessage from 'components/generalErrorMessage/generalErrorMessage';
+import type { RenderPrbInput } from 'components/StripePaymentRequestButton/StripePaymentRequestButton';
+import StripePaymentRequestButtonContainer from 'components/StripePaymentRequestButton/StripePaymentRequestButtonContainer';
+import type {
+	ContributionType,
+	OtherAmounts,
+	SelectedAmounts,
+} from 'helpers/contributions';
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import { fromCountry } from 'helpers/internationalisation/countryGroup';
+import type { IsoCurrency } from 'helpers/internationalisation/currency';
+
+interface StripePaymentRequestButtonProps {
+	country: IsoCountry;
+	currency: IsoCurrency;
+	isTestUser: boolean;
+	contributionType: ContributionType;
+	selectedAmounts: SelectedAmounts;
+	otherAmounts: OtherAmounts;
+}
+
+function StripePaymentRequestButton({
+	country,
+	currency,
+	isTestUser,
+	contributionType,
+	selectedAmounts,
+	otherAmounts,
+}: StripePaymentRequestButtonProps): JSX.Element {
+	return (
+		<StripePaymentRequestButtonContainer
+			country={country}
+			currency={currency}
+			isTestUser={isTestUser}
+			contributionType={contributionType}
+			selectedAmounts={selectedAmounts}
+			otherAmounts={otherAmounts}
+			renderPrb={({
+				type,
+				paymentRequest,
+				paymentRequestError,
+				onStripePrbClick,
+				onCustomPrbClick,
+			}) => (
+				<Prb
+					country={country}
+					contributionType={contributionType}
+					type={type}
+					paymentRequest={paymentRequest}
+					paymentRequestError={paymentRequestError}
+					onStripePrbClick={onStripePrbClick}
+					onCustomPrbClick={onCustomPrbClick}
+				/>
+			)}
+		/>
+	);
+}
+
+// ---- Helper component ---- //
+
+interface PrbProps extends RenderPrbInput {
```

Comment:
Is it lazy to just extend this type here? it's convenient but I don't feel amazing about it